src/tables.js

```diff
@@ -228,18 +230,18 @@ Flow.integerToNote.table = {
 };
 
 
-Flow.tabToGlyph = fret => {
+Flow.tabToGlyph = (fret, scale) => {
```

Comment:
Since you're changing the `tabToGlyph` signature here, you'll need to set a default value for scale (so we don't break existing users.)